src/libraries/System.Text.Json/gen/TypeGenerationSpec.cs

```diff
@@ -58,6 +58,10 @@ internal class TypeGenerationSpec
 
         public TypeGenerationSpec? NullableUnderlyingTypeMetadata { get; private set; }
 
+        public string? RuntimeTypeRef { get; private set; }
```

Comment:
It might make sense to stick a comment on this property explaining what it is and what it is used for. At first read, I don't really understand what a `RuntimeTypeRef` is.

Reply:
I added a comment:

```
/// <summary>
/// Supports deserialization of extension data dictionaries typed as I[ReadOnly]Dictionary<string, object/JsonElement>.
/// Specifies a concrete type to instanciate, which would be Dictionary<string, object/JsonElement>.
/// </summary>
```

Reply:
I'm assuming this would not be used if the extension data property is of type `JsonObject`, right?